base/src/lib/export-schema.ts

```diff
@@ -213,7 +213,17 @@ async function exportToTypescript(
           break
         }
       }
+      case "string": {
+        const containsEnums: boolean = definition?.enum
```

Comment:
`definition` is not optional. No need for `?`
```suggestion
        const containsEnums: boolean = definition.enum
```

Reply:
To reduce code, maybe just put this check inside the parenthesis of the if

Reply:
okay